src/sagemaker/estimator.py

```diff
@@ -2219,7 +2219,20 @@ def _validate_and_set_debugger_configs(self):
         ):
             self.debugger_hook_config = DebuggerHookConfig(s3_output_path=self.output_path)
         elif not self.debugger_hook_config:
```

Comment:
Just to understand it, so it's possible for `self.debugger_hook_config` to be `None` or `False`. And you're changing it to `False` intentionally to disable it?

Reply:
The default value of `self.debugger_hook_config` is None.
The if-statement above checks if the debugger is supported in the region in which it is invoked and if not it should be set to `False` instead of `None` in my opinion.